spidermon/contrib/actions/slack/__init__.py

```diff
@@ -92,8 +92,8 @@ def _get_users_info(self):
             ]
         )
 
-    def _api_call(self, method, **kwargs):
-        response = self._client.api_call(method, **kwargs)
+    def _api_call(self, method, json=None):
+        response = self._client.api_call(method, json=json)
```

Comment:
**suggestion**: According to [migration guide](https://github.com/slackapi/python-slackclient/wiki/Migrating-to-2.x#web-client-api-changes) in the example code, the use of `api_call` method is still allowed, but the most efficient way (as they mentioned) is to use `chat_postMessage` call. Perhaps it would be better to stop using `api_call` and using it for all requests we make (I think it would affect calls to `conversations.open` and `users.list`) following the library recommendation.